.circleci/config.yml

```diff
@@ -353,9 +351,13 @@ workflows:
           dist: centos
           release: "7"
       - distcheck:
-          name: distcheck_centos_8
-          dist: centos
-          release: "8"
+          name: distcheck_fedora_stable
+          dist: fedora
+          release: latest
+      - distcheck:
+          name: distcheck_fedora_rawhide
+          dist: fedora
+          release: rawhide
```

Comment:
I would prefer `distcheck_fedora_latest` to match the image tag.

Reply:
That would be smarter indeed